### PR TITLE
Added MC truth track parameters

### PR DESCRIPTION
**Pull request by sophiemiddleton**
test with https://github.com/Mu2e/Offline/pull/1182

### CODE REVIEW

File: fcl/prolog_trigger.fcl [hunk 1]
```diff
@@ -14,7 +14,7 @@ TrkAnaTrigger : {
     FillCaloMC : false
     RecoCountTag : ""
     FillTriggerInfo : true
-    FillTrkQualInfo : false
+    //FillTrkQualInfo : false
```
Comment (AndrewEdmonds11) [inline fcl/prolog_trigger.fcl:17]: You can just remove this line - the parameter no longer exists. I missed deleting it here when doing the recent TrkQual update

**Review by AndrewEdmonds11 — APPROVED** [review]
Thanks, Sophie. This looks great. Just one small comment to delete a fcl parameter that no longer exists. As far as I can tell, this doesn't need to wait for the Offline and Production PRs, right?

**Comment by sophiemiddleton** [comment]
I think it needs to wait for the Offline Pr

File: src/InfoMCStructHelper.cc [hunk 2]
```diff
@@ -209,12 +232,13 @@ namespace mu2e {
     siminfo.valid = true;
     if(sp.genParticle().isNonnull())siminfo.gen = sp.genParticle()->generatorId().id();
     siminfo.proc = sp.creationCode();
+    siminfo.stop = sp.stoppingCode();
```
Comment (AndrewEdmonds11) [inline src/InfoMCStructHelper.cc:235]: Hi Sophie - I don't see a ```stop``` leaf in the ```SimInfo``` struct in this PR. Did you meain to include an updated SimInfo.hh?

**Pull request merged**